### DISCUSSION TITLE
tips for host optimization

### DISCUSSION THREAD

**Discussion by orcutt989** [86Box/86Box#3066] [Q&A] [answered]
I have a i9 13900k running at 5.6Ghz and I can barely run a P2 at 200Mhz.  I know that 86Box is not VMware. I keep seeing discussions about "single thread performance" and "host clock speed", but I feel like I should be able to run over a 500Mhz processor on my setup.  

Are there any tips like setting process affinity, etc that I am missing? I use WinBox to keep my builds up to date daily.  Should I roll back to only use stable build? I have tried both new dynamic recompiler and old compiler.

**Comment by jriwanek** [comment, marked as answer]
Make sure you set the HDD to something other than RAM disk, use Old Dynarec, not new, and switching to ymfm from nuked helps too.

I'm on a 12700k myself, and last I tested, I could manage 233-266 reliably.

**Reply by orcutt989** [reply]
Thanks for the tips! 

With Win98 I am able to do a 333Mhz Via Cyrix III stably. However with Win XP it seems to not be stable at 200Mhz. I just borked my machine by switching motherboards so I'll have to do it again and test.
